framework/rest/UrlRule.php

```diff
@@ -244,17 +246,33 @@ public function parseRequest($manager, $request)
      */
     public function createUrl($manager, $route, $params)
     {
+        $this->createStatus = WebUrlRule::CREATE_STATUS_SUCCESS;
         foreach ($this->controller as $urlName => $controller) {
             if (strpos($route, $controller) !== false) {
                 foreach ($this->rules[$urlName] as $rule) {
-                    /* @var $rule \yii\web\UrlRule */
+                    /* @var $rule WebUrlRule */
                     if (($url = $rule->createUrl($manager, $route, $params)) !== false) {
+                        $this->createStatus = WebUrlRule::CREATE_STATUS_SUCCESS;
                         return $url;
+                    } elseif (
+                        $this->createStatus === null
+                        || !method_exists($rule, 'getCreateUrlStatus')
```

Comment:
Yes. `UrlRuleInterface` does not contain this method, so you can't be sure that it exist.